theme_notes/views/snippets/s_product_list.xml

```diff
@@ -4,68 +4,68 @@
 <template id="s_product_list" inherit_id="website.s_product_list">
```

Comment:
Same comments than for the bookstore theme => this is not achievable with the editor :grimacing: Keep it simple like the original snippet :slightly_smiling_face: